service/controller/v1/prometheus/scrapeconfig.go

```diff
@@ -362,6 +372,32 @@ func getScrapeConfigs(service v1.Service, certificateDirectory string) []config.
 			},
 		},
 	}
+	// check if we can add etcd monitoring
+	if _, ok := service.Labels[key.LabelVersionBundle]; ok {
```

Comment:
Doesn't it make more sense to perform that check on `AnnotationEtcdDomain`?

Reply:
Yep, i will do that.